tox.ini

```diff
@@ -1,26 +1,11 @@
 [tox]
-envlist = py35,py36,py37,py38,lint,check-formatting,manifest
+envlist = py35,py36,py37,py38,manifest
 isolated_build = True
 
 [testenv]
 extras = tests
 commands = python -m pytest {posargs}
 
-[testenv:lint]
-basepython = python3.8
-deps =
-    pre-commit
-commands = pre-commit run --all-files
```

Comment:
I think I'm okay removing `check-formatting` and `format`, but I'm against removing this entirely. I _hate_ git pre-commit hooks, and very much prefer `tox -e lint`. Since CI already has a pre-commit, you can just remove `lint` from the `envlist` but still keep this here.

Reply:
Ok, sure I know the good old muscle memory problem and here speaks nothing against keeping the command around 😉
I trained myself/shell history to use `pre-commit run -a` now and so often. ^^